packages/admin/cms-admin/src/warnings/warningsConfig.ts

```diff
@@ -0,0 +1,14 @@
+import { type ReactNode } from "react";
+
+import { useCometConfig } from "../config/CometConfigContext";
+import { warningMessages as cometWarningMessages } from "./warningMessages";
+
+export interface WarningsConfig {
+    messages: Record<string, ReactNode>;
+}
+
+export function useWarningConfig(): WarningsConfig {
```

Comment:
Naming: useWarningsConfig. Sorry, missed that in the first round of review.

Reply:
a0ab2b7500635a00261d49e1d87f96857942c9ff